packages/signature-controller/src/types.ts

```diff
@@ -0,0 +1,37 @@
+// Trace types later will be imported from '@metamask/controller-utils'
+/** A context in which to execute a trace, in order to generate nested timings. */
+export type TraceContext = unknown;
```

Comment:
We can remove all these duplicate types in this PR since we can reference them in `controller-utils`.

All that means is when we come to release, we release a new `controller-utils` at the same time.

Reply:
Sorry not sure I follow, how can we reference from `controller-utils` in this PR without breaking this PR build?

Reply:
Because the build understands these dependencies are local so it will use the local code.

Reply:
Ok that's what was trying to ask earlier in the slack thread, so the build understands but we need to update dependencies at the release branch. Now all settled :)